### ISSUE TITLE
Log level setup option.

### ISSUE THREAD

**Issue by xszuflax** [tomekkleszcz/homebridge-electrolux-devices#27]
**Is your feature request related to a problem? Please describe:**
At the moment plugin floods HomeBridge logs with constant:

```
[12/10/2023, 13:16:32] [HomebridgeElectroluxDevices] Polling appliances status...
[12/10/2023, 13:16:33] [HomebridgeElectroluxDevices] Appliances status polled!
```

**Describe the solution you'd like:**
It would be nice to have log level setup so that only warnings or errors are logged for stable setups.  Or  debug level if one is playing with the configuration. 

**Describe alternatives you've considered:**
N/A

**Additional context:**
N/A

Thank you for your great work!

**Comment by tomekkleszcz** [comment]
@xszuflax I've changed the log level to debug for these messages. Should be resolved in #34

**Comment by xszuflax** [comment]
Hello @tomekkleszcz ,

fix is good :-) thank you very much!
